.travis.yml

```diff
@@ -10,6 +10,7 @@ install:
     - pip install -r requirements-dev.txt
 script:
     - make test_coverage
+    - flake8 --version
```

Comment:
This doesn't seem necessary?

Reply:
I needed this one to find out what version of flake you use so I could fix the build

Reply:
I need this so I could find out what version of flake you need so I could fix the flake errors